### PR TITLE
Feature-final touches

### PR DESCRIPTION
**Pull request by maximoortelli**
At this point i should have a fully working app that uses only basic styles to make the layout work, according to this wireframe:

leaderboard_wireframe

- [x] I should improve the look and feel of the application, adding the styles of my choice.

- [x] I keep the general layout of the wireframe, this is the only mandatory requirement.

- [x] I can use plain CSS or any CSS framework, i decide.

### CODE REVIEW

File: src/style.css
Comment (Efiamotu-1) [inline src/style.css:1]: - [x] From the project wireframe as shown below, your design for the form is not aligned properly, the button is not properly aligned with the input fields as shown in your design image below, please kindly align your design properly and resubmit for approval 👍 
- project wireframe
fw
- your design
nal

File: README.md
Comment (Efiamotu-1) [inline README.md:1]: - [x] Your readme file is missing two important sections which are the `install & usage` sections please see the [microverse readme template](https://github.com/microverseinc/readme-template) for  reference and add the missing sections

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @maximoortelli  

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style.css
Comment (maximoortelli) [reply src/style.css:1]: Thank you, done👍😃

File: README.md
Comment (maximoortelli) [reply README.md:1]: Thank you, done👍😃

**Review by OLIPLICHE — APPROVED** [review]
Hi @maximoortelli, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights
- The App is styled and matches the wireframe (✅)) 👏
- Applied what was requested by the previous review 👏
- The App is working fine👏 👏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**